redbot/core/cli.py

```diff
@@ -237,6 +238,15 @@ def parse_cli_flags(args):
     parser.add_argument(
         "--no-message-cache", action="store_true", help="Disable the internal message cache."
     )
+    parser.add_argument(
+        "--disable-intent",
```

Comment:
If this is set this way are we going to be applying all intents by default?

Reply:
Yes.

Reply:
I guess it makes sense since the majority of users are under 100 servers, but given that majority of users also dont read. I would honestly feel more confortable if this was off by default and its a new flag we tell users to start red with based on what they have enabled.

Reply:
The current way this PR works Draper is it ensure the bot will at least start for most of our users. If we don't have all intents enabled once the bot starts reaching more guilds it will increase start time for many users.

Reply:
But the intent are all off by default and need to be enabled via the portal, this is what im referencing most users not reading.

I expect the vast majority of live users today to not have the intents enabled in the portal.

Reply:
The error for when people try to use privileged intents will contain information on how to enable them and I will most likely also update docs appropriately. While I do want to add some functionality related to intents (be it checks, metadata or whatever TBD), currently `--disable-intent` is just there so people with big bots can at least disable the intents. Currently all 1st-party and 3rd-party cogs expect to receive all events and it does not really make sense to change the default and bother a big majority of users to change the default.

I just want to remind this is purposefully just a draft, and the better errors as well as doc changes are still to be done.

Reply:
Something to note is that currently this does not allow disabling members or presences intent.

Reply:
That's not true, `**dict.fromkeys(cli_flags.disable_intent, False)` that is lower in the dict literal will 
 just replace them.